migrations/20170419122022_make-is_primary-not-nullable.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+exports.up = (knex) => (
+  knex.schema
+    .raw('UPDATE records SET is_primary = false where is_primary IS NULL')
+    .raw('ALTER TABLE records ALTER COLUMN is_primary SET NOT NULL')
```

Comment:
Went for a raw query on altering the table, mostly because the `knex` variant seemed way more complex and harder to read. At least coming after the value alteration above.